### ISSUE TITLE
Neopets Advert Attack game not working

### ISSUE THREAD

**Issue by ostentia** [ruffle-rs/ruffle#10561]
Hi,

The Advert Attack game is not working in Safari, Chrome, or Firefox. The "Go" icon on the bottom of the screen is supposed to move every time it gets clicked on, but it gets stuck after being clicked only once or twice. This makes the game unplayable. Can you please fix this? Thank you :)

**Comment by n0samu** [comment]
Duplicate of #7660

**Comment by n0samu** [comment]
I looked at this issue recently and from what I remember, I wasn't able to make any progress in finding the cause.

**Comment by ostentia** [comment]
Ah, bummer! Thank you for getting back to me and for all of the hard work you've done on the site.
